code/game/turfs/simulated/footsteps.dm

```diff
@@ -125,6 +125,9 @@
 	if(step_count % step_interval)
 		return
 
+	if(istype(move_intent, /decl/move_intent/stalk)) //We don't make sounds if we're tiptoeing
```

Comment:
this is sloppy, add a flag to move intents indicating that they don't make sounds instead,, or specifying a really long sound interval